InAppUtils/InAppUtils.m

```diff
@@ -117,12 +117,15 @@ - (void)paymentQueueRestoreCompletedTransactionsFinished:(SKPaymentQueue *)queue
         NSMutableArray *productsArrayForJS = [NSMutableArray array];
         for(SKPaymentTransaction *transaction in queue.transactions){
             if(transaction.transactionState == SKPaymentTransactionStateRestored) {
-              NSDictionary *purchase = @{
-                @"originalTransactionIdentifier": transaction.originalTransaction.transactionIdentifier,
-                @"transactionIdentifier": transaction.transactionIdentifier,
-                @"productIdentifier": transaction.payment.productIdentifier,
-                @"transactionReceipt": [[transaction transactionReceipt] base64EncodedStringWithOptions:0]
-              };
+                SKPaymentTransaction *originalTransaction = transaction.originalTransaction;
+
+                NSDictionary *purchase = @{
+                    @"originalTransactionDate": @(originalTransaction.transactionDate.timeIntervalSince1970 * 1000),
```

Comment:
yeah. Sorry that RCTConvert usage was incorrect. I meant to say convert NSDate to string and send date as a string to JS.